dcpy/library/templates/sca_enrollment_capacity.yml

```diff
@@ -3,7 +3,7 @@ dataset:
   acl: public-read
   source:
     socrata:
-      uid: 8b9a-pywy
+      uid: gkd7-3vk7
```

Comment:
Was the old one retired, or is this just a slightly different/better source?

Reply:
seems to be retired ([link](https://data.cityofnewyork.us/d/8b9a-pywy/)). if you login with our admin account, it exists but it's private. so seems they've replaced it